frontend/packages/client/src/components/CommunityProposals.js

```diff
@@ -5,11 +5,14 @@ import { useCommunityProposalsWithVotes, useMediaQuery } from "../hooks";
 import { FilterValues } from "../const";
 import DropDownFilter from "./ProposalsList/DropdownFilter";
 import WrapperResponsive from "./WrapperResponsive";
+import { omit } from "lodash";
 
 export default function CommunityProposals({ community = { id: 1 } }) {
   const notMobile = useMediaQuery();
 
-  const proposalFilterValues = Object.values(FilterValues);
+  const proposalFilterValues = Object.values(
+    omit(FilterValues, ["inprogress", "terminated"])
+  );
```

Comment:
any specific reason for using `omit` vs using native `filter` here?

Reply:
using filter instead